fern/docs.yml

```diff
@@ -6,64 +6,193 @@ instances:
+            path: ./overview/dummy.mdx
+          - page: Schema Validation
+            path: ./overview/dummy.mdx
+          - page: Auto-Pagination
+            path: ./overview/dummy.mdx
+          - page: Retries with backoff
+            path: ./overview/dummy.mdx
+          - page: Webhook Signature Verification
+            path: ./overview/dummy.mdx
+          - page: Forward Compatibility
+            path: ./overview/dummy.mdx
+          - page: Multipart Form Data
+            path: ./overview/dummy.mdx
+          - page: Idempotency Headers
+            path: ./overview/dummy.mdx
+          - page: Server-Sent Events
+            path: ./overview/dummy.mdx
+          - page: Integration Tests
+            path: ./overview/dummy.mdx
+          - page: Code Snippets
+            path: ./overview/dummy.mdx
+          - page: Augment with custom code
+            path: ./overview/dummy.mdx
+          - page: Merging multiple APIs
+            path: ./overview/dummy.mdx
+          - page: GitHub integration
+            path: ./overview/dummy.mdx
+  - tab: docs
+    layout:
+      - page: Overview
+        path: ./overview/dummy.mdx
+      - page: Quickstart
+        path: ./overview/dummy.mdx
+      - section: Global Settings
+        collapsed: true
+        contents:
+          - page: URLs
+            path: ./overview/dummy.mdx
+            slug: urls
+          - page: Colors
+            path: ./overview/dummy.mdx
+          - page: Layout
+            path: ./overview/dummy.mdx
+          - page: Navigation
+            path: ./overview/dummy.mdx
+          - page: Typography
+            path: ./overview/dummy.mdx
+          - page: CSS and JS Overrides
+            path: ./overview/dummy.mdx
+          - page: Call to actions
+            path: ./overview/dummy.mdx
+      - section: Writing Content
+        collapsed: true
+        contents:
+          - page: Page Titles and Metadata
+            path: ./overview/dummy.mdx
+          - page: Headers and Text
+            path: ./overview/dummy.mdx
+          - page: Images, Videos and Embeds
+            path: ./overview/dummy.mdx
+          - page: Lists and Tables
+            path: ./overview/dummy.mdx
+          - page: Code Blocks
+            path: ./overview/dummy.mdx
+      - section: API References
+        collapsed: true
+        contents:
+          - page: API Definition Setup
+            path: ./overview/dummy.mdx
+          - page: Errors, Audiences, and Snippets
+            path: ./overview/dummy.mdx
+          - page: API Playground
+            path: ./overview/dummy.mdx
+      - section: Custom Components
+        collapsed: true
+        contents:
+          - page: Accordion
+            path: ./overview/dummy.mdx
+          - page: Availability
+            path: ./overview/dummy.mdx
+          - page: Callout
+            path: ./overview/dummy.mdx
+          - page: Cards
+            path: ./overview/dummy.mdx
+          - page: Code Blocks
```

Comment:
```suggestion
          - page: CodeBlocks
```

Reply:
Not only because the component is named CodeBlocks, but also to avoid confusing with the `Code Blocks` page under the Writing Content section.